deployment/testing-v4-gcloud-self/README.md

```diff
@@ -0,0 +1,167 @@
+# Testing configuration for v4 containers
+
+This directory contains terraform configuration to bring relevant resources for
+an integration test of the prio-processor v4.x containers.
+
+To create a new project that uses the same configuration, change the terraform
+backend appropriately. Here, the state is placed into a storage bucket that has
+been created beforehand. Ensure the project has also been created. Then:
```

Comment:
I think you could outline the steps here more clearly. I assume you mean a GCP project? And that it should be created before the storage bucket? Or is the storage bucket created by terraform?